test/resources/openshift_auth.go

```diff
@@ -4,6 +4,7 @@ import (
 	goctx "context"
 	"fmt"
 	"github.com/PuerkitoBio/goquery"
+	"golang.org/x/net/context"
```

Comment:
use goctx instead

Reply:
done

Reply:
@MStokluska , thank you for review and that you found that "context" is dupplicated in many places. 
I did search and updated all required. No need add every one in review, as it's a bit not easy, just maybe say that need check all required places. But anyway, all updated, and thanks that you found it.

Reply:
I'm not doing it to just make it easier for you, but also for myself.
Sometimes owner of PR can mention that something has been updated while in fact it wasn't. Mentioning it on every occurrence saves me going through all the files over again, and instead I can rely on on gh resolving the thread if it's outdated. 

For example, here: https://github.com/integr8ly/integreatly-operator/pull/3460/files#diff-632835e0eac94542c834d7b410205dd7cd7719641144b67580295278dd8e3643R181 you have mentioned it's done, while I can see it's not.